src/operator/numpy/np_elemwise_broadcast_op.cc

```diff
@@ -198,5 +198,74 @@ MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_npi_minimum_scalar)
 .set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Compute<cpu, mshadow_op::minimum>)
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseIn{"_backward_minimum_scalar"});
 
+inline bool Arctan2OpType(const nnvm::NodeAttrs& attrs,
+	                          std::vector<int>* in_attrs,
+	                          std::vector<int>* out_attrs){
+     CHECK_EQ(in_attrs->size(), 2U);
+     CHECK_EQ(out_attrs->size(), 1U);
+
+     TYPE_ASSIGN_CHECK(*out_attrs, 0, in_attrs->at(0));
+     TYPE_ASSIGN_CHECK(*out_attrs, 0, in_attrs->at(1));
+     TYPE_ASSIGN_CHECK(*in_attrs, 0, out_attrs->at(0));
+     TYPE_ASSIGN_CHECK(*in_attrs, 1, out_attrs->at(0));
+     //check if it is float16, float32 or float64. If not, raise error
+     if(in_attrs->at(0) > 2){
+     // do not support int now.
+     std::ostringstream os;
+     os << "Do not support `int` as input.\n";
+     throw ::mxnet::op::InferTypeError(os.str(), 0);
+     }
+     return out_attrs->at(0) != -1;
```

Comment:
2-space indentations, please fix for all your code

Reply:
Fixed, thank you.